### ISSUE TITLE
Another Boot loop on Zero W doesn't find wpa_supplicant

### ISSUE THREAD

**Issue by AnotherBantar** [ccrisan/motioneyeos#1797]
My PC is windows 10, I'm using the Pi Zero W.  I have tried this version 
motioneyeos-raspberrypi-dev20190130.img.xz
 and the one created before it several times.  

I use SD Formatter to format my micro SD, then Win32DiskImager to write the disc image.  Then I follow the directions on the wiki page and create the txt file wpa_supplicant.conf in the only directory I can find files on windows explorer for the SD card.  I then safely remove and plug the card into my Pi Zero W and it boots and gets to wired connection and doesn't find a device and reboots over and over.  It never uses the wpa_supplicant.  Any ideas what I'm doing wrong?  Thanks so much!  -John

**Comment by ccrisan** [comment]
1. Are you sure your `wpa_supplicant.conf` file name is correct?
2. Have you tried the same with previous stable versions?

**Comment by ccrisan** [comment]
20180627 is the latest stable version. Also have you checked out the [wifi troubleshooting](https://github.com/ccrisan/motioneyeos/wiki/Troubleshooting#wireless-connection-problems) article?

**Comment by AnotherBantar** [comment]
Wow.  It seems I'm an idiot.  I just bought this computer and am not at all used to windows 10.  When I would create the file wpa_supplicant.conf it never added the .txt.  BUT I didn't know that windows 10 has an option to hide all of the extensions.  I just looked at the file type after I made it for the 800th time and saw that it was still a text file.  When I clicked on "show file extensions" the .txt popped right up.  Deleting that worked like a charm.  Sorry, it's been a long couple days but thanks very much for the help!
